distribution/openrct2.d.ts

```diff
@@ -599,14 +603,90 @@ declare global {
      * Represents a guest or staff member.
      */
     interface Peep extends Entity {
+        /**
+         * Name of the peep.
+         */
+        name : string;
+
         /**
          * Colour of the peep's t-shirt.
          */
         tshirtColour: number;
+
         /**
          * Colour of the peep's trousers.
          */
         trousersColour: number;
+
+        /**
+         * How tired the guest is between 32 and 128 where lower is more tired.
+         */
+        energy : number;
+
+        /**
+         * The target energy value. Energy will increase / decrease slowly towards this value.
+         */
+        energyTarget : number;
+
+        /**
+         * How happy the guest is between 0 and 255.
+         */
+        happiness : number;
+
+        /**
+         * The target happiness value. Happiness will increase / decrease slowly towards this value.
+         */
+        happinessTarget : number;
+
+        /**
+         * How nauseated the guest is between 0 and 255.
+         */
+        nausea : number;
+
+        /**
+         * The target nausea value. Nausea will increase / decrease slowly towards this value.
+         */
+        nauseaTarget : number;
+
+        /**
+         * How hungry the guest is between 0 and 255. Lower is more hungry.
+         */
+        hunger: number;
```

Comment:
A minor nitpick, but you seem to use inconsistent formatting (you use both `hunger: number;` and ``hunger : number;`).